policyengine_us/parameters/gov/states/nd/tax/income/taxable_income/subtractions/ltcg_fraction.yaml

```diff
@@ -1,14 +1,17 @@
 description: North Dakota subtracts this fraction of long-term capital gains from US taxable income when calculating its taxable income.
 values:
   2021-01-01: 0.40
-  2022-01-01: 0.40
   
 metadata:
-  label: fraction of long-term capital gains that can be subtracted from taxable income
+  label: North Dakota fraction of long-term capital gains that can be subtracted from taxable income
   unit: /1
   period: year
   reference:
-  - title: 2021 ND income tax form and instruction booklet, line 7
-    href: https://www.tax.nd.gov/sites/www/files/documents/forms/individual/2021-iit/individual-income-tax-booklet-2021.pdf #page12
-  - title: 2022 ND income tax form and instruction booklet, line 7
-    href: https://www.tax.nd.gov/sites/www/files/documents/forms/2022-individual-income-tax-booklet.pdf #page12
+  - title: 2021 North Dakota income tax form and instruction booklet, line 7
+    href: https://www.tax.nd.gov/sites/www/files/documents/forms/individual/2021-iit/individual-income-tax-booklet-2021.pdf#page=12
+  - title: 2022 North Dakota income tax form and instruction booklet, line 7
+    href: https://www.tax.nd.gov/sites/www/files/documents/forms/individual/2022-iit/2022-individual-income-tax-booklet.pdf#page=12
+  - title: 2023 North Dakota income tax form and instruction booklet, line 6
+    href: https://www.tax.nd.gov/sites/www/files/documents/forms/individual/2023-iit/2023-individual-income-tax-booklet.pdf#page=14
+  - title: North Dakota legal code Section 57-38-30.3 2(d)(1)
```

Comment:
I dont think this section mentions the 40% - please double check

Reply:
Double check all newly adde legal code to make sure that the number is actually referenced in the section